ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+P3A.swift

```diff
@@ -14,6 +14,35 @@ import os.log
 
 extension BrowserViewController {
 
+  func recordWeeklyUsage() {
+    var weeklyUsage = P3ATimedStorage<Int>.weeklyUsage
+    weeklyUsage.replaceTodaysRecordsIfLargest(value: 1)
+    let calendar = Calendar(identifier: .gregorian)
+    let mondayWeekday = 2
+    let isTodayMonday = calendar.component(.weekday, from: .now) == mondayWeekday
+    guard
+      let thisMonday = isTodayMonday
+        ? calendar.startOfDay(for: .now)
+        : calendar.nextDate(
+          after: .now,
+          matching: .init(weekday: mondayWeekday),
+          matchingPolicy: .nextTime,
+          direction: .backward
+        ),
+      let lastMonday = calendar.nextDate(
+        after: thisMonday,
+        matching: .init(weekday: mondayWeekday),
+        matchingPolicy: .nextTime,
+        direction: .backward
+      )
+    else {
+      return
+    }
+    let answer = weeklyUsage.combinedValue(in: lastMonday..<thisMonday)
```

Comment:
is `lastMonday` inclusive and `thisMonday` exclusive?

Reply:
Yep! the monday itself wouldn't be counted, the added unit test has a range test that looks like this too, to verify